xds/annotations/v3/status.proto

```diff
@@ -4,6 +4,8 @@ package xds.annotations.v3;
 
 import "google/protobuf/descriptor.proto";
 
+option go_package = "github.com/cncf/xds/go/xds/annotations/v3";
```

Comment:
I'm not actually sure if this should include the second xds or not, in the other case I omitted this, but can't recall why